build.sbt

```diff
@@ -68,9 +64,14 @@ lazy val munit = project
     libraryDependencies ++= List(
       "junit" % "junit" % "4.13",
       "com.geirsson" % "junit-interface" % "0.11.6",
-      "com.googlecode.java-diff-utils" % "diffutils" % "1.3.0",
-      "org.scala-lang" % "scala-reflect" % scalaVersion.value % Provided
-    )
+      "com.googlecode.java-diff-utils" % "diffutils" % "1.3.0"
+    ) ++ {
+      scalaBinaryVersion.value match {
+        case "0.21" => Nil
+        case _ =>
+          List("org.scala-lang" % "scala-reflect" % scalaVersion.value)
```

Comment:
I couldn't get the tests to pass with `Provided`, this is probably something bad in my setup